geodesy/package.xml

```diff
@@ -25,17 +26,24 @@
 
   <run_depend>geographic_msgs</run_depend>
   <run_depend>geometry_msgs</run_depend>
-  <run_depend>python-pyproj</run_depend>
   <run_depend>tf</run_depend>
   <run_depend>sensor_msgs</run_depend>
   <run_depend>unique_id</run_depend>
   <run_depend>uuid_msgs</run_depend>
 
+  <run_depend condition="$ROS_PYTHON_VERSION == 2">python-pyproj</run_depend>
+  <run_depend condition="$ROS_PYTHON_VERSION == 3">python3-pyproj</run_depend>
+
+  <buildtool_depend condition="$ROS_PYTHON_VERSION == 2">python-setuptools</buildtool_depend>
+  <buildtool_depend condition="$ROS_PYTHON_VERSION == 3">python3-setuptools</buildtool_depend>
+  <buildtool_export_depend condition="$ROS_PYTHON_VERSION == 3">python3-setuptools</buildtool_export_depend>
```

Comment:
No need to `buildtool_export_depend` setuptools. `buildtool_depend` is correct because it's needed to build this package, but it's not needed by downstream packages when they build.

Reply:
Got it, will do.